src/screens/DiscoveryScreenV2/fragment/DomainFragment.js

```diff
@@ -325,7 +325,7 @@ const styles = StyleSheet.create({
     fontFamily: fonts.inter[600]
   },
   unfollowedHeaderContainer: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Similarly, the change in `backgroundColor` for `unfollowedHeaderContainer` style from `COLORS.lightgrey` to `COLORS.gray100` should be reviewed for consistency with the app's theme and user interface.

```diff
-    backgroundColor: COLORS.lightgrey
+    backgroundColor: COLORS.gray100
```
Please ensure these color changes have been thoroughly tested across different screens and components for consistency and usability.


<!-- This is an auto-generated comment by OSS CodeRabbit -->